lib/cesium_link_web/live/link_live/form_component.ex

```diff
@@ -19,6 +19,7 @@ defmodule CesiumLinkWeb.LinkLive.FormComponent do
         <.input field={@form[:emoji]} type="emoji" label="Emoji" />
         <.input field={@form[:url]} type="text" label="URL" />
         <.input field={@form[:attention]} type="checkbox" label="Attention" />
+        <.input field={@form[:publish_at]} type="datetime-local" label="Publish At" />
```

Comment:
Can you make it so that the scheduling time input only shows up when you are creating a new link? 💪